dist/nearlib.js

```diff
@@ -350,6 +350,8 @@ const {
 
 const MAX_STATUS_POLL_ATTEMPTS = 10;
 const STATUS_POLL_PERIOD_MS = 2000;
+// Default amount to be send with the function calls.
```

Comment:
what does this "amount" mean? what's going to happen to it?

Reply:
Added more explanation.